client/src/scrollableItems.tsx

```diff
@@ -35,16 +36,12 @@ interface ScrollableItemsProps {
 const isScrollbarVisible = (scrollableArea: HTMLDivElement) =>
   scrollableArea.scrollHeight > scrollableArea.clientHeight;
 
-const elementIsVisible = (
-  scrollableArea: HTMLDivElement,
-  element: HTMLElement
+const isScrolledToBottom = (
+  { scrollTop, scrollHeight, offsetHeight }: HTMLDivElement,
+  hasUnread: boolean
 ) => {
-  const elementTopRelativeToScrollableArea =
-    element.offsetTop - scrollableArea.offsetTop;
-  const scrollTop = scrollableArea.scrollTop;
-  const scrollableAreaHeight = scrollableArea.clientHeight;
-  const scrollTopThreshold =
-    elementTopRelativeToScrollableArea - scrollableAreaHeight - 10;
+  const maxScrollTop = scrollHeight - offsetHeight - (hasUnread ? 32 : 0);
+  const scrollTopThreshold = maxScrollTop - 10;
```

Comment:
addressed by moving the height of the 'jump to bottom' button to a const, so now this ternary says `(hasUnread ? JUMP_TO_BOTTOM_HEIGHT : 0)` to avoid having a comment (this is obvs re-used in the css of the 'jump to bottom' button' too)